src/main/java/com/bitso/Bitso.java

```diff
@@ -174,6 +178,41 @@ public BitsoOpenOrders getOpenOrders(BitsoBook book) {
     }
 
     /**
+     * @param The
+     *            specific endpoint to get from a {@link com.bitso.exchange.BitsoLedgerEntry}.
+     * @return The {@link com.bitso.exchange.BitsoLedgerEntry} of the user.
+     */
```

Comment:
+Intente varias opciones como pasar únicamente el {@link..} a otro renglón, pasar el enunciado a otro renglón después del @param, inclusive revise las opciones del más actual bitso-java-code-format.xml como:
<setting id="org.eclipse.jdt.core.formatter.insert_new_line_after_annotation_on_parameter" value="do not insert"/>
Pero aparentemente está bien seteado y aún sigue haciendo el resultado obtenido tras presionar Command + Shift + f